docs/process/releasing-updates.md

```diff
@@ -12,12 +12,77 @@ We have three channels to which we can release: `production`, `beta`, and `test`
 
 ## The Process
 
-1. Ensure the release notes for `$version` in [`changelog.json`](../../changelog.json) are up-to-date.
-1. Bump `version` in [`app/package.json`](../../app/package.json) to `$version`.
-1. Commit & push the changes.
-1. Run `.release! desktop/YOUR_BRANCH to {production|beta|test}`.
-  * We're using `.release` with a bang so that we don't have to wait for any current CI on the branch to finish. This might feel a little wrong, but it's OK since making the release itself will also run CI.
-1. If you're releasing a production update, release a beta update for the next version too, so that beta users are on the latest release.
+From a clean working directory, set the `GITHUB_ACCESS_TOKEN` environment variable to a valid [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) and run:
+
+```shellsession
+$ yarn draft-release (production|beta)
+```
```

Comment:
For a one-liner, `GITHUB_ACCESS_TOKEN=$(cat) yarn draft-release (production|beta)`, hit enter, paste your token, hit enter, then hit `^D`.

```shellsession
$ FOO=$(cat) bash
foo-bar
^D
> echo $FOO
foo-bar
```

Reply:
I left this here because it's totally going to fail in any reasonable shell, but I'm open to a simpler way to demonstrate the options available

Reply:
```shellsession
$ yarn draft-release production
or:
$ yarn draft-release beta
```

Reply:
Actually, to come back to the earlier point:

> For a one-liner, `GITHUB_ACCESS_TOKEN=$(cat) yarn draft-release (production|beta)`, hit enter, ...

Let's not make this shell-specific. I call out the environment variable in the previous paragraph, and that's applicable to all OSes.

But I'll tweak this sample to show these `production` and `beta` inputs as separate statements...